openvdb/tools/ParticlesToLevelSet.h

```diff
@@ -414,7 +414,7 @@ template<typename SdfGridT, typename AttributeT, typename InterrupterT>
 template<typename ParticleListT, typename GridT>
 struct ParticlesToLevelSet<SdfGridT, AttributeT, InterrupterT>::Raster
 {
-    typedef typename boost::is_void<AttributeT>::type DisableT;
+    typedef typename std::is_void<AttributeT>::type DisableT;
```

Comment:
Does this work? Looks like the boost version is either `true_type` or `false_type` whereas the std version is a value, `true` or `false`.
I think we aren't targeting C++17 yet, so sadly we can't use the `_v` and `_t` suffixes on these std metafunctions.

Reply:
I think the std::void () operator is a value, but ::type as far as I know is a true_type/false_type still (https://en.cppreference.com/w/cpp/types/is_void). If I'm wrong, then happy to be told so :)